sonar-core/src/main/java/org/sonar/core/util/stream/MoreCollectors.java

```diff
@@ -110,6 +112,13 @@ private MoreCollectors() {
       ImmutableSet::copyOf);
   }
 
+  /**
+   * A Collector into an {@link EnumSet} of specified enumeration.
+   */
+  public static <E extends Enum<E>> Collector<E, ?, EnumSet<E>> toEnumSet(Class<E> enumClass) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove usage of generic wildcard type. ([squid:S1452](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1452))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2FSONAR-10117%2Flive-measures&issues=AWBujJM-mGlYLmlUfGDK&open=AWBujJM-mGlYLmlUfGDK)

[ISSUE_KEY]: AWBujJM-mGlYLmlUfGDK

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED